packages/nuxt/src/head/module.ts

```diff
@@ -54,9 +54,24 @@ export default defineNuxtModule({
       addPlugin({ src: resolve(runtimeDir, 'plugins/vueuse-head-polyfill') })
     }
 
-    if (nuxt.options.experimental.headCapoPlugin) {
-      addPlugin({ src: resolve(runtimeDir, 'plugins/capo') })
-    }
+    addTemplate({
+      filename: 'unhead-plugins.mjs',
+      getContents () {
+        if (!nuxt.options.experimental.headNext) {
+          return 'export default []'
+        }
+        return `import { CapoPlugin, HashHydrationPlugin } from '@unhead/vue'
+       export default [
+         CapoPlugin({ track: true }),
```

Comment:
do we need this plugin on client-side?

Reply:
Oh yeah, that thought did cross my mind, outside of SPAs, I can't imagine it's that useful.

I'll see what I can do

Reply:
Hm also doesn't really make sense to use HashHydration for SPA templates either, is there an environment variable I can use for this?

Otherwise will need to move this code to the renderer and the unhead plugin.